### ISSUE TITLE
Is I have to copy the libc source code in the same direction, then I can write the hello world program?

### ISSUE THREAD

**Issue by hstk30** [rui314/chibicc#91]
I write the hello world, want to try it

```
#include <stdio.h>

int main() {
    printf("hello world!");
    return 0;
}
```

and output error

```
hello.c:1: #include <stdio.h>
                    ^ stdio.h: cannot open file: No such file or directory
```

So, how can I run it successfully?

**Comment by wangjinnana** [comment]
now it may has solved, in commit a939a7a90638631c296dfb63d857b24555b25327 the author add function add_default_include_paths, add default include path /usr/local/include,/usr/include/x86_64-linux-gnu,/usr/include

**Comment by hstk30** [comment]
Ok, I see that code. I think maybe my OS is MacOS Monterey, which the libc path is not in that default path. 

I run `hello world` in centos is successful.
